hw2/04-background.js

```diff
@@ -1 +1,27 @@
 // Add your code here
+
+const colors = ['#FF0000', '#FFA500', '#FFFF00', '#00FF00', '#0000FF', '#A020F0', '#FFC0CB']
+
+async function startColorChange() 
+{
+   let interval = document.getElementById("interval");
+
+   document.getElementById("start").hidden = true;
+   document.getElementById("stop").hidden = false;
+
+   while(document.getElementById("stop").hidden == false)
+   {
+        document.body.style.backgroundColor = colors[Math.floor(Math.random() * 8)];
+        await sleep(interval.value);
+   }
```

Comment:
There is no sort of validation being performed on the value of the interval. Meaning that the user can enter 0 or a negative number, thus cause the program to misbehave.